src/Raven.Server/Documents/DocumentPutAction.cs

```diff
@@ -130,7 +130,9 @@ protected virtual void ValidateId(Slice lowerId)
             id = BuildDocumentId(id, newEtag, out bool knownNewId);
             using (DocumentIdWorker.GetLowerIdSliceAndStorageKey(context, id, out Slice lowerId, out Slice idPtr))
             {
-                ValidateId(lowerId);
+                if (flags.HasFlag(DocumentFlags.FromResharding) == false && 
+                    nonPersistentFlags.HasFlag(NonPersistentDocumentFlags.FromSmuggler) == false)
```

Comment:
why we do not want to validate items from Smuggler? You are assuming that they are correct?